src/lib/components/HttpErrors/HttpError.js

```diff
@@ -4,7 +4,7 @@ import { Grid, Button, Icon } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
 import Overridable from 'react-overridable';
 
-class HttpError extends Component {
+export class HttpError extends Component {
```

Comment:
the class is already exported from this file (check the last line of this file)

Reply:
i know, but it does not work me if i do not export it like this

Reply:
I understand but it is not a correct solution, double export is misleading